Feedback

x21L opened the pull request:
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>


Subscribed: @Thomas-Witt

x21L committed:
Setting up GitHub Classroom Feedback

Thomas-Witt committed:
let it chiffre at random or choose a number. you can even choose ridiculous high (like 1.000.000), or even negative numbers. Spaces won't be affected.

Thomas-Witt committed:
let it chiffre at random or choose a number. you can even choose ridijust ned to re-commit this, because I didn't check the boxes.culous high (like 1.000.000), or even negative numbers. Spaces won't be affected. Again,

gerhofer commented on scr/CaesarChiffre.java:
```diff
@@ -1,31 +1,45 @@
+        System.out.println("Should it switch at random (press 1) or do you want to choose (press 2)");
+        int choice = scanner.nextInt();
+        int charjump;
+        if (choice == 2) {
```
cool 😎

gerhofer commented on scr/CaesarChiffre.java:
```diff
@@ -1,31 +1,45 @@
+        charjump = charjump % 26;
+        char[] characters = new char[message.length()];
+
+        for (int i = 0; i < message.length(); i++) {
```
genau diese Funktionalität gibt es zum glück bereits 😃 
siehe https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/lang/String.html#toCharArray()

gerhofer commented on scr/CaesarChiffre.java:
```diff
@@ -1,31 +1,45 @@
-            } else {
-                changedLetter[i] = (char) (singleLetter[i] - 26 + characterJump);
+        for (int i = 0; i < message.length(); i++) {
+            if ((int) characters[i] != 32) {
```
anstelle hier die numerische repräsentation zu verwenden fänd ichs viel lesbarer, wenn du einfach direkt die characters verwenden würdest. 

also ich fänd `if (characters[i] != ' ')` viel lesbarer, das gleiche gilt für die ganzen zahlen unten

Thomas-Witt committed:
the method "toCharArray()" saves 2 lines (and makes it easier to read),
using '' do directly write the letter instead of its numeric code also makes the code prettier

gerhofer reviewed (approved):
cool find ich jetzt super lesbar 👍
